lang/java/wiredtiger.i

```diff
@@ -362,7 +362,7 @@ javaClose(JNIEnv *env, WT_SESSION *session, JAVA_CALLBACK *jcb, jfieldID *pfid)
 
 	(*env)->SetLongField(env, jcb->jobj, fid, 0L);
 	(*env)->DeleteGlobalRef(env, jcb->jobj);
-	__wt_free(jcb->session, jcb);
+	free(jcb);
```

Comment:
This change may be OK since the memory is completely owned by Java. On the other hand, if there is a custom memory allocator configured in WiredTiger, I'd rather the Java language binding uses the same allocator rather than the system allocator.